conversations/v3/telegram.yaml

```diff
@@ -113,7 +113,7 @@ paths:
         - "Callback configuration"
       summary: Update the callback
       operationId: updateWeChatChannelCallback
-      description: Update the callback settings of a specific WeChat ID.
+      description: Update the callback settings of a specific Telegram Bot ID.
```

Comment:
Shouldn't the operationId be also updated to `updateTelegramChannelCallback`?

Reply:
Who knows? :shrug: Changed... https://github.com/radekholy24/tyntec-api-collection/pull/157/commits/db3c61c7f7f459092c427bea3701ce3188fc9e7c